contracts/base/EthOracle.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity ^0.4.11;
+
+import "../Oracle/PriceOracleInterface.sol";
+import "./AuctioneerManaged.sol";
+import "./DxMath.sol";
+
+contract EthOracle is AuctioneerManaged, DxMath {
+    uint constant WAITING_PERIOD_CHANGE_ORACLE = 30 days;
+    
+    // Price Oracle interface 
+    PriceOracleInterface public ethUSDOracle;
+    // Price Oracle interface proposals during update process
+    PriceOracleInterface public newProposalEthUSDOracle;
+    
+    uint public oracleInterfaceCountdown;
+
+    event NewOracleProposal(
+         PriceOracleInterface priceOracleInterface
+    );
+
+    function initiateEthUsdOracleUpdate(
+        PriceOracleInterface _ethUSDOracle
+    )
+        public
+        onlyAuctioneer
+    {         
+        require(address(_ethUSDOracle) != address(0));
```

Comment:
Maybe you can add an error message here.

Reply:
Thanks, will do!